docs/quickstart.rst

```diff
@@ -2,54 +2,17 @@
 Quick start
 =============
 
-Example locustfile.py
```

Comment:
Why remove "example" from this headline?

Reply:
Because it is more clear that the file name is important. But you are right, it was better before. I'll add it back in.